RecoLocalCalo/HcalRecAlgos/src/PulseShapeFitOOTPileupCorrection.cc

```diff
@@ -406,10 +411,10 @@ int PulseShapeFitOOTPileupCorrection::pulseShapeFit(const double * energyArr, co
    bool  fitStatus   = false;
    bool useTriple = false;
 
-   int BX[3] = {4,5,3};
+   int BX[3] = {int(soi),int(soi+1),int(soi-1)};
```

Comment:
Are these castings needed? I.e. couldn't BX remain a vector of unsigned ints, also in the fit(..) method

Reply:
fixed int vs unsigned